telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
+message NameIDMessage { //Send the device ID with its corresponding name
+	optional uint32 id = 1;
+	optional string name = 2 [(nanopb).max_size = 32];
+}
+
+message StatisticalAggregate {
+  optional uint32 numSamples = 1;
+  optional uint32 min = 2;
+  optional uint32 max = 3;
+  optional uint32 mean = 4;
+  optional uint32 stdev = 5;
+}
+
+message GeneralReading { //this might want a more specific name?
+	enum READING_TYPE {
+		VOLTAGE = 1;
+		CURRENT = 2;
+		TEMPERATURE = 3;
+	}
+	optional READING_TYPE type = 1;
```

Comment:
Would we want to/be able to calculate the first four metadata? Resolution, accuracy, and the sensor limits seem like inherent components of the sensor, so could we just include it in the config file that was being discussed in the electrical-fw chat?

Also, you brought up a good point for units, since right now everything is being sent as a uint32 in the proto, would it make more sense to just define some smallest unit either universally or in the config file to keep our readings as integers (millivolts or milliamps, for example)? Sending some two-character string for the units every time seems a bit unnecessary, unless I'm missing something.